src/App.vue

```diff
@@ -18,7 +18,13 @@
 				/>
 			</template>
 
-			<template #footer>
+			<AppHelp v-if="!loading.notes && helpVisible" :settingsOpen="helpVisible" @popupClosed="helpVisible = false" />
+			<template slot="footer">
```

Comment:
Why not keeping this like before?
```suggestion
			<template #footer>
```